scripts/bm-report/report.qmd

```diff
@@ -0,0 +1,314 @@
+---
+title: "Velox Build Metrics"
+execute:
+  echo: false
+  warning: false
+format:
+  html:
+    grid:
+        sidebar-width: 0px
+        body-width: 1800px
+        margin-width: 150px
+        gutter-width: 1.5rem
+    self-contained: true
+    page-layout: full
+    toc: false
+    margin-left: 30px
+    link-external-newwindow: true
+    theme: cosmo
+---
+
+
+
+
+```{r setup}
+library(gt)
+library(ggplot2)
+library(plotly)
+library(dplyr)
+library(purrr)
+
+cd <- cachem::cache_disk(rappdirs::user_cache_dir("velox-bm-report"))
```

Comment:
I suppose this is R code, can you add a brief comment on whats happening here :)

Reply:
I have added some general comments as reference but didn't go into to much detail because there is nothing out of the ordinary in there in terms of R code.